bciers/libs/utils/src/formatTimestamp.ts

```diff
@@ -0,0 +1,24 @@
+const formatTimestamp = (timestamp: string) => {
```

Comment:
This is a copy over from reg1, not a refactor. I left reg1 using its own version of this function